Tooling/requirements.txt

```diff
@@ -0,0 +1,2 @@
+sdc11073
+unittest-xml-reporting
```

Comment:
i think we need to fix the cryptography version, i got issues when installing without a fixed version.

Reply:
nevermind, with a new pip version it's all good.